website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/authenticate-azure.md

```diff
@@ -5,7 +5,7 @@ description: "dbt Cloud developers need to authenticate with Azure DevOps."
 sidebar_label: "Authenticate with Azure DevOps"
 ---
 
-If you are a dbt Cloud developer who contributes code to the dbt Cloud IDE, and you use Azure DevOps for collaborating on code and building and deploying apps, you need to [link your dbt Cloud profile to Azure DevOps](#link-your-dbt-cloud-profile-to-azure-devops). Linking these two profiles provides access to your Azure DevOps profile from your dbt Cloud profile by authenticating.
+If you use the dbt Cloud IDE to collaborate on your team's Azure DevOps dbt repo, you need to [link your dbt Cloud profile to Azure DevOps](#link-your-dbt-cloud-profile-to-azure-devops), providing an extra layer of authentication.
```

Comment:
@schottj I like how this now focuses on the task they do and now include more than "developers." Just a tiny suggestion:

```suggestion
If you use the dbt Cloud IDE to collaborate on your team's Azure DevOps dbt repo, you need to [link your dbt Cloud profile to Azure DevOps](#link-your-dbt-cloud-profile-to-azure-devops), which provides an extra layer of authentication.
```